Sources/Jobs/JobInstance.swift

```diff
@@ -29,7 +29,7 @@ public protocol JobInstanceProtocol: Sendable {
     /// Time job was queued
     var queuedAt: Date { get }
     /// Number of attempts so far
-    var attempts: Int? { get }
+    var attempts: Int { get }
```

Comment:
Can we rename this to `attempt` as it indicates the attempt number now, and not the number of attempts that have already been made. Also edit the comment above to indicate the change in meaning.